tests/unit_tests/service/test_interface.py

```diff
@@ -307,9 +307,9 @@ def test_cancel_active_task(cancel_active_task_mock: MagicMock):
 @patch("blueapi.service.interface.TaskWorker.get_tasks")
 def test_get_tasks(get_tasks_mock: MagicMock):
     tasks = [
-        TrackableTask(task_id="0", task=None),
-        TrackableTask(task_id="1", task=None),
-        TrackableTask(task_id="2", task=None),
+        TrackableTask(task_id="0", task=Task(name="0")),
+        TrackableTask(task_id="1", task=Task(name="1")),
+        TrackableTask(task_id="2", task=Task(name="1")),
```

Comment:
```suggestion
        TrackableTask(task_id="2", task=Task(name="2")),
```

nit, just to make it clearer what is going on if the test breaks